backend/test/til_web/controllers/post_controller_test.exs

```diff
@@ -27,6 +27,66 @@ defmodule TilWeb.PostControllerTest do
       assert first_post["categoriesIds"] == [first_category.id, second_category.id]
       assert second_post["categoriesIds"] == [first_category.id]
     end
+
+    test "returns all existing posts with proper like count", %{conn: conn} do
+      current_user = insert(:user)
+
+      first_post = insert(:post)
+      second_post = insert(:post)
+
+      first_like = insert(:like, user_id: current_user.id, post_id: first_post.id)
+      second_like = insert(:like, user_id: current_user.id, post_id: first_post.id)
```

Comment:
technically this should not be allowed (unique index should prevent duplicate records on the same user/post pair)